test/functional/JIT_Test/src/jit/test/tr/chtable/VirtualGuardTest.java

```diff
@@ -91,7 +95,7 @@ private char bar() {
 	}
 	
 	private int goo() {
-		return (f.method() + f.method() + f.method() + f.method());
+                return (f.method() + f.method() + f.method() + f.method());
```

Comment:
For some reason the tabs got replaced with spaces here making it inconsistent with the rest of the file.

Reply:
Fixed it !